odoo/addons/base/data/ir_module_category_data.xml

```diff
@@ -129,6 +129,16 @@
             <field name="sequence">50</field>
         </record>
 
+        <record model="ir.module.category" id="module_category_esg">
+            <field name="name">ESG</field>
+            <field name="sequence">52</field>
+        </record>
+
+        <record model="ir.module.category" id="module_category_esg_esg">
```

Comment:
Why add this?
I don't see where it's used. I recommend discussing this with @cth-odoo , as it's not going in the intended direction.